src/pages/workspace/tags/WorkspaceTagsPage.tsx

```diff
@@ -267,6 +287,8 @@ function WorkspaceTagsPage({route}: WorkspaceTagsPageProps) {
         setIsDeleteTagsConfirmModalVisible(false);
     };
 
+    const selectedTagsObject = selectedTagsArray.map((key) => sortedTags.find((tag) => tag.name === key));
```

Comment:
Move inside `getHeaderButtons`

Reply:
Also after using `policyTagLists.at(0)?.tags` we won't need to use `.find` as we can access the object values by keys directly